source/_components/fibaro.markdown

```diff
@@ -25,22 +26,32 @@ fibaro:
   url: http://192.168.1.161/api/
   username: your_username
   password: your_password
+  plugins: true or false (default false)
```

Comment:
The first config example should be minimal, with only required items.

Reply:
Removed.